src/backend/web/graphql/index.js

```diff
@@ -0,0 +1,109 @@
+const { gql: graphql } = require('apollo-server-express');
```

Comment:
Sorry to make you change this if it's not their standard way.  That said, I think line 15 reads better from a maintenance point of view, with devs who aren't as familiar with graphql, and will get a better sense of what is going on.  Thanks for doing this.